extra/resources/ClusterMon

```diff
@@ -153,7 +153,8 @@ ClusterMon_monitor() {
     if [ -f $OCF_RESKEY_pidfile ]; then
 	pid=`cat $OCF_RESKEY_pidfile`
 	if [ ! -z $pid ]; then
-	    kill -s 0 $pid >/dev/null 2>&1; rc=$?
+	    ps -wwfp $pid | grep $OCF_RESKEY_pidfile >/dev/null 2>&1 && \
+	        kill -s 0 $pid >/dev/null 2>&1; rc=$?
```

Comment:
As discussed off-GH, COLUMNS doesn't imply automic enlargement of _field_ width.  This should be better in that regard:

```
str=$(echo "su - $OCF_RESKEY_user -c "${HA_SBIN_DIR}/crm_mon -p $OCF_RESKEY_pidfile EXTRA SPACIOUS RESERVE" | tr 'crmon, \t' 'xxxxxxxx')
ps ... -o "args=${str}"
```